resources/i18n/ru/cura.po

```diff
@@ -0,0 +1,3546 @@
+msgctxt "@title"
+msgid "Build Plate Leveling"
+msgstr "Выравнивание стола"
+
+#: /home/ruben/Projects/Cura/plugins/UltimakerMachineActions/BedLevelMachineAction.qml:38
+msgctxt "@label"
+msgid ""
+"To make sure your prints will come out great, you can now adjust your "
+"buildplate. When you click 'Move to Next Position' the nozzle will move to "
+"the different positions that can be adjusted."
+msgstr ""
+"Сейчас вы можете отрегулировать ваш стол, чтобы быть уверенным в качестве "
+"последующей печати. При нажатии на кнопку «Перейти к следующей позиции» "
+"сопло перейдёт на другую позиции, которую можно будет отрегулировать."
+
+#: /home/ruben/Projects/Cura/plugins/UltimakerMachineActions/BedLevelMachineAction.qml:47
+msgctxt "@label"
+msgid ""
+"For every position; insert a piece of paper under the nozzle and adjust the "
+"print build plate height. The print build plate height is right when the "
+"paper is slightly gripped by the tip of the nozzle."
+msgstr ""
+"Для каждой позиции, вставьте кусок бумаги под сопло и отрегулируйте высоту "
+"стола. Когда кончик сопла немного прижимает бумагу к столу, значит вы "
+"выставили правильную высоту стола."
+
+#: /home/ruben/Projects/Cura/plugins/UltimakerMachineActions/BedLevelMachineAction.qml:62
+msgctxt "@action:button"
+msgid "Start Build Plate Leveling"
+msgstr "Начало выравнивания стола"
+
+#: /home/ruben/Projects/Cura/plugins/UltimakerMachineActions/BedLevelMachineAction.qml:74
+msgctxt "@action:button"
+msgid "Move to Next Position"
+msgstr "Перейти к следующей позиции"
+
+#: /home/ruben/Projects/Cura/plugins/UltimakerMachineActions/UpgradeFirmwareMachineAction.qml:27
+msgctxt "@title"
+msgid "Upgrade Firmware"
+msgstr "Обновление прошивки"
+
+#: /home/ruben/Projects/Cura/plugins/UltimakerMachineActions/UpgradeFirmwareMachineAction.qml:38
+msgctxt "@label"
+msgid ""
+"Firmware is the piece of software running directly on your 3D printer. This "
+"firmware controls the step motors, regulates the temperature and ultimately "
+"makes your printer work."
+msgstr ""
+"Прошивка является программным обеспечением, которое работает на плате вашего "
+"3D принтера. Прошивка управляет шаговыми моторами, регулирует температуру и, "
+"в конечном счёте, обеспечивает работу вашего принтера."
+
+#: /home/ruben/Projects/Cura/plugins/UltimakerMachineActions/UpgradeFirmwareMachineAction.qml:48
+msgctxt "@label"
+msgid ""
+"The firmware shipping with new printers works, but new versions tend to have "
+"more features and improvements."
+msgstr ""
+"Прошивка, поставляемая с новыми Ultimaker, работает, но обновления "
+"предоставляют больше возможностей и усовершенствований."
+
+#: /home/ruben/Projects/Cura/plugins/UltimakerMachineActions/UpgradeFirmwareMachineAction.qml:62
+msgctxt "@action:button"
+msgid "Automatically upgrade Firmware"
+msgstr "Автоматическое обновление прошивки"
+
+#: /home/ruben/Projects/Cura/plugins/UltimakerMachineActions/Upgr
```

Comment:
It's missing a few of the ampersands here, but it could be that these only work for Latin characters. And in any case that's not a blocking problem.